### ISSUE TITLE
DevInfoBanner window uses client side config instead of server side

### ISSUE THREAD

**Issue by AruMoon** [space-wizards/space-station-14#13728]
## Description
The DevInfoBanner window, which contains a link to the Bug report, uses the client_config.toml, which does not contain any info links or may contain old info links that are no longer updated because those CVars are now server-side. The bug_report setting in the server config is currently useless because of this. It only works if you add a default CVar value to generic CVars.cs, i.e. just hard code

**Reproduction**
- Build server
- Setup infolinks.bug_report cvar in server_config.toml
- Run server and connect
- No bug report button

**Screenshots**

![изображение](https://user-images.githubusercontent.com/28903731/214767014-0157c531-4386-4aeb-89b6-0f18ed336302.png)

**Comment by VerinSenpai** [comment]
I don't believe this is an issue anymore. The CVar for InfoLinksBugReport is now server and replicated. See https://github.com/space-wizards/space-station-14/blob/e7be57d85f8f0644aac9010378d35f5cbc6c4224/Content.Shared/CCVar/CCVars.Game.Infolinks.cs#L46

**Comment by VerinSenpai** [comment]
Perhaps my confidence should not be so fragile 😅
